### ISSUE TITLE
Dropping Python 3.9?

### ISSUE THREAD

**Issue by hassec** [cfs-energy/cfspopcon#105]
I'd like to understand if anybody is relying on python 3.9 compatibility.

Numpy v2.1 and later as well as the latest xarray release have dropped python 3.9 and require >=3.10

We'll eventually have to follow this to be able to update these dependencies. This isn't urgent, but I'd rather raise this point early to already incentivize people to use python 3.10 and later.  

fyi: @tbody-cfs @nelsonand @AudreySaltzman (feel free to tag more people that might be affected)

**Comment by tbody-cfs** [comment]
Not using 3.9 and happy to drop

**Comment by nelsonand** [comment]
Also happy to drop!

**Comment by AudreySaltzman** [comment]
Yep - I don't use Python 3.9 and I checked with the rest of MIT folks - we're all good to drop 3.9.

**Comment by hassec** [comment]
Thanks everyone for the quick round of replies! ❤️
